charts/landscaper/gardenlet/garden-cluster-rbac/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
What's this? All of these things are already part of the Gardener `controlplane` chart, right? Why do we duplicate this now here?

Reply:
These are only the RBAC roles required in the Garden cluster for the gardenlet to work.

> are already part of the Gardener controlplane

In my point of view this does not matter as the Gardenlet landscaper component is meant to be an independently installable unit that does not rely on some RBAC rules created in another chart. 
In fact I would argue these RBAC rules do not semantically belong to the `controlplane` chart - they are only relevant for Gardenlets and gardenlets are not part of the control plane.

Reply:
This also holds true in my point of view for the VPA CRD. if that is required in the runtime cluster, we should install it.

Reply:
No, sorry, this component is not meant to be independent of the world. It clearly is depending on other things, most prominently (and to stay in the scope of this comment) on the Gardener control plane. You can deploy whatever RBAC roles you want into the garden cluster - if there is no Gardener API Server, etc. then your installable gardenlet unit will not work.

Eventually, the to-be-introduced new landscape-setup (that, among others, includes this gardenlet component) is the thing that you consider "independently installable unit" (though, even that's not 100% correct because it will still depend on container registries, a first root cluster, etc.). This component is one puzzle piece that should be limited in its scope. The composition of those components is what will be delivered.

Reply:
> No, sorry, this component is not meant to be independent of the world.

I think I did not express myself clearly above.
Sure the Gardelet is dependent on a running control plane.

By independently _installable unit_  I rather meant to say that the Gardenlet component should contain all the resources that are only required by the Gardenlet or that are needed to set it up.
And the gardenlet RBAC roles in the Garden cluster only belong to the Gardenlet and not to the control plane chart. The Control plane can run without it, the Gardenlet cannot.

Additionally, contrary to my [own source code comment below ](https://github.com/gardener/gardener/pull/3130#discussion_r523926747) the gardenlet component can control the whole lifecycle of these RBAC roles. 
They are always created in the garden cluster and only removed when there are no more Seed clusters in the system.

Reply:
lets involve some more colleagues @timebertt @schrodit @BeckerMax @timuthy .
In the end I am fine with either ways. Just want to make sure that we involved the right people and make an informed decision.
Thanks @rfranzke

Reply:
I don't get this discussion, TBH.
FMPOV, this RBAC stuff is belonging to the controlplane stuff (as it is already the case today), as it should be managed centrally (only once per gardener installation and not per instance of the gardenlet component).
It is required for having a running gardener installation in which you can register Seeds/gardenlets, similar to how the kube-apiserver ensures the default clusterroles (including `system:node` and `system:node-bootstrapper`) in it's PostStartHooks, ref https://github.com/kubernetes/kubernetes/blob/f57611970c0790b325719e0279bfae334537e2de/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L188